main_test.go

```diff
@@ -53,6 +61,29 @@ func TestSendMetrics(t *testing.T) {
 	assert.NoError(err)
 }
 
+func TestSendMetricsHostStripping(t *testing.T) {
+	for _, tt := range hostStrippingDataSet {
+		assert := assert.New(t)
+		event := corev2.FixtureEvent(tt.entityName, "check1")
+		event.Check = nil
+		event.Metrics = corev2.FixtureMetrics()
+		event.Metrics.Points[0].Name = tt.entityName + ".answer"
+
+		var apiStub = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			body, _ := ioutil.ReadAll(r.Body)
+			assert.Contains(string(body), tt.expectedBody)
+			w.WriteHeader(http.StatusOK)
+			_, err := w.Write([]byte(`{"ok": true}`))
+			require.NoError(t, err)
+		}))
+
+		config.Addr = apiStub.URL
+		config.StripHost = true
```

Comment:
If you make this field configurable in the test case, you can set `config.StripHost` like this!
```suggestion
		config.StripHost = tt.stripHost
```